docs/UserGuide.md

```diff
@@ -289,12 +289,31 @@ Want to get rid of the meeting link under a particular tag? Use the `linkdelete`
 Step 1: First key in `taglist` and hit enter to display all the tags.
 ![LinkDeleteCommand Step 1](images/LinkDeleteCommandGuide1.png)
 
-Step 2: For example, if you want to remove the tag from the "classmates" tag, key in `linkdelete 3` and hit enter.
+Step 2: For example, if you want to remove the link from the "classmates" tag, key in `linkdelete 3` and hit enter.
 ![LinkDeleteCommand Step 2](images/LinkDeleteCommandGuide2.png)
 
 Step 3: You should see the link being removed from the "classmates" tag like this.
 ![LinkDeleteCommand Step 3](images/LinkDeleteCommandGuide3.png)
 
+#### Marking a task under a tag as done: `taskdone`
+
+After adding task to your targeted tag, you must have seen the cross beside the task description. So how to change it 
+to a tick so that you can keep track of the task progress? Use `taskdone` followed by the index `INDEX` of the tag and 
+the alphabetical index `ALPHAINDEX` to do the magic!
```

Comment:
Potential errors/fixes:
- after adding `a` task...
- maybe it would be more succinct to say "so how to mark the task as done once you've completed the task"